src/main/zapHomeFiles/hud/libs/i18n.js

```diff
@@ -0,0 +1,256 @@
+import Vue from "vue";
+import VueI18n from "vue-i18n";
+
+const messages = {
+  en_GB: {
+    message: {
+      alerts_all_title: "All Alerts",
+      alerts_details_title: "Alert Details",
+      alerts_field_attack: "Attack",
+      alerts_field_confidence: "Confidence",
+      alerts_field_cweid: "CWE Id",
+      alerts_field_description: "Description",
+      alerts_field_evidence: "Evidence",
+      alerts_field_other: "Other Info",
+      alerts_field_parameter: "Parameter",
+      alerts_field_reference: "Reference",
+      alerts_field_risk: "Risk",
+      alerts_field_solution: "Solution",
+      alerts_field_url: "URL",
+      alerts_field_wascid: "WASC Id",
+      alerts_page_high_tool: "Page Alerts: High",
+      alerts_page_info_tool: "Page Alerts: Info",
+      alerts_page_low_tool: "Page Alerts: Low",
+      alerts_page_medium_tool: "Page Alerts: Medium",
+      alerts_page_title: "Page Alerts",
+      alerts_risk_high: "High",
+      alerts_risk_info: "Informational",
+      alerts_risk_low: "Low",
+      alerts_risk_medium: "Medium",
+      alerts_site_high_tool: "Site Alerts: High",
+      alerts_site_info_tool: "Site Alerts: Info",
+      alerts_site_low_tool: "Site Alerts: Low",
+      alerts_site_medium_tool: "Site Alerts: Medium	",
```

Comment:
The trailing whitespace here is suspicious

Reply:
Note that this file is generated (from `UIMessages.properties`), these comments apply to develop.

Reply:
Moved to #650